packages/onboarding/src/utils/flows.ts

```diff
@@ -112,12 +111,8 @@ export const isStartWritingFlow = ( flowName: string | null ) => {
 	return Boolean( flowName && [ START_WRITING_FLOW ].includes( flowName ) );
 };
 
-export const isDesignFirstFlow = ( flowName: string | null ) => {
-	return Boolean( flowName && [ DESIGN_FIRST_FLOW ].includes( flowName ) );
-};
-
 export const isBlogOnboardingFlow = ( flowName: string | null ) => {
```

Comment:
We can probably do a follow-up PR to use `isStartWritingFlow` and remove `isOnboardingFlow` since they will be equivalent.